inst/rmarkdown/templates/transmissibility/skeleton/skeleton.Rmd

```diff
@@ -19,9 +19,9 @@ params:
   use_epiparameter: 
```

Comment:
Could you also rename this param since we're now using epiparameter anyways? Maybe something like `use_epiparamer_db` could work? Happy to follow your naming suggestion.

Reply:
I agree, I'll update it to `use_epiparameter_database` ? to make it 100% clear